src/firestore/firekit.ts

```diff
@@ -144,6 +145,13 @@ interface LevanteSurveyResponses {
   [key: string]: string;
 }
 
+interface UpdateTaskVariantData {
+  taskId: string;
+  taskData: FirestoreTaskData;
+  variantId?: string;
+  variantData?: FirestoreVariantData;
+}
```

Comment:
Since you'll only ever need the variant data OR the task data, maybe we could just supply one or the other.
```suggestion
interface UpdateTaskVariantData {
  taskId: string;
  data: FirestoreTaskData | FirestoreVariantData;
  variantId?: string;
}
```